pkg/services/ngalert/metrics/ngalert.go

```diff
@@ -163,24 +166,47 @@ func newSchedulerMetrics(r prometheus.Registerer) *Scheduler {
 			},
 			[]string{"org"},
 		),
-		GetAlertRulesDuration: promauto.With(r).NewHistogram(
+		SchedulePeriodicDuration: promauto.With(r).NewHistogram(
 			prometheus.HistogramOpts{
 				Namespace: Namespace,
 				Subsystem: Subsystem,
-				Name:      "get_alert_rules_duration_seconds",
-				Help:      "The time taken to get all alert rules.",
+				Name:      "schedule_periodic_duration_seconds",
+				Help:      "The time taken to run the scheduler.",
 				Buckets:   []float64{0.1, 0.25, 0.5, 1, 2, 5, 10},
 			},
 		),
-		SchedulePeriodicDuration: promauto.With(r).NewHistogram(
+		SchedulableAlertRules: promauto.With(r).NewGauge(
+			prometheus.GaugeOpts{
+				Namespace: Namespace,
+				Subsystem: Subsystem,
+				Name:      "schedulable_alert_rules",
```

Comment:
When looking at metrics you typically look by the prefix and is strange that metrics that are closely correlated have different (but a similar variation of the word schedule) and appear inconsistent. 

How do you feel about using the word of the component "schedule" as the prefix for all of them?

Reply:
Should we use `scheduler` as the prefix?

Reply:
I have a slight preference for the word `schedule` instead of the component `scheduler` - the latter is most likely to change than the former. Up to you really.